src/Engine/MapController.java

```diff
@@ -186,7 +186,10 @@ private void handleMapInput(InputHandler.InputAction input)
       case SEEK: // Move the cursor to the next unit that is ready to move.
```

Comment:
This comment should be updated to reflect that we cycle through both active units and usable properties.